### ISSUE TITLE
Thank You form button needs more contrast

### ISSUE THREAD

**Issue by youriwims** [mozilla/donate-wagtail#322]
### Issue: [Elements must have sufficient color contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=AxeChrome)

**Element path:** `.button--secondary`

**Snippet:** `<a class="button button--large button--secondary" href="/en-US/thank-you/">No Thanks</a>`
<img width="501" alt="Screen Shot 2019-09-30 at 7 57 02 PM" src="https://user-images.githubusercontent.com/36484608/65924922-c6ca1500-e3bc-11e9-9958-ce1f69b590e8.png">

**How to fix:**
Fix any of the following:
- Element has insufficient color contrast of 2.84 (foreground color: #ffffff, background color: #999999, font size: 13.5pt, font weight: bold). Expected contrast ratio of 4.5:1

====

More Info:

Comments: This is also a concern in upsell view[#328].
Resources: n/a

**Comment by nicklee** [comment]
@beccaklam do you want to provide us with a new colour to replace the grey?

**Comment by beccaklam** [comment]
@nicklee We can change it to #666666 and ensure that the text is bold and at least 18px

**Comment by danielfmiranda** [comment]
After working together with pomax, we have found that the button has been updated to the latest style guidelines:

<img width="497" alt="image" src="https://user-images.githubusercontent.com/18314510/112364701-0745de80-8c94-11eb-9c54-38b57040804a.png">

We then checked to make sure the button met accessibility standards, and it did! So we are closing the issue
